### ISSUE TITLE
Depends on java-11 when java-17 installed

### ISSUE THREAD

**Issue by roman4e** [hvdwolf/jExifToolGUI#234]
**Version (please complete the following information):**
 - OS: Ubuntu
 - Version 20.04
 - Package deb

**Describe the bug**
I'm trying to install latest package, but it depends on openjre-11 however I have oracle jdk-17 installed.

**To Reproduce**
1. install oracle java 17
2. Download latest package
3. run sudo dpkg -i ${latest_package_name}.deb
 dpkg: dependency problems prevent configuration of jexiftoolgui:
 jexiftoolgui depends on openjdk-11-jre; however:
  Package openjdk-11-jre is not installed.
  Package oracle-java11-installer-local which provides openjdk-11-jre is not installed.

It has no sense to install java-11 when I have another newer version installed.

**Comment by hvdwolf** [comment]
I fully agree with you. I changed the dependency to >=1.11. This weekend I will try to release a 2.0.1 bug release

**Comment by hvdwolf** [comment]
Please try [this beta 2.0.1](https://mega.nz/file/lNsCiQbZ#A-qN5z-McOOK5V8nfrclUKWPuk2V9DiIhfBgofkJLl8).
It should use V11 or newer.

**Comment by roman4e** [comment]
Cool. Thank you.
